clients/go/apiban/apiban.go

```diff
@@ -233,7 +234,7 @@ func ProcBannedResponse(entry *Entry, id string, blset ipset.IPSet) {
 		//os.Exit(0)
 	}
 
-	ttl := GetConfig().blTtl
+	ttl := int(GetConfig().BlacklistTtl / time.Second) // round-down to seconds
 	if ttl == 0 {
```

Comment:
Note: ttl from config can never be 0 now (the config loading will fail if blacklist ttl == 0).
Should we allow it?

Reply:
yes, we need to since the config ttl overrides the metadata configuration ttl.